test_wazuh/test_fim/test_duplicate_entries/test_duplicate_entries.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2015-2019, Wazuh Inc.
```

Comment:
This module should be place under test_fim/test_ambiguous_confs/test_duplicate_entries.py